### DISCUSSION TITLE
Game Lore by Clyde

### DISCUSSION THREAD

**Discussion by aeolus-1** [aeolus-1/triangleGame.github.io#47] [Announcements]
Once upon a time, there were two shapes - cubes and circles. The cubes were always trying to be the best shape possible, while the circles just wanted to have fun and enjoy life. The cubes would try to fit into every nook and cranny they could find, while the circles would lazily float around in open space. The cube's quest for perfection led them to become very stiff and inflexible, whereas the circle's carefree attitude allowed them to remain fluid and ever-changing. Over time, people began preferring the flexibility of Circles over the rigidity of Cubes - leading Cube factories to go out of business whilst Circle businesses thrived!

The story of the cubes and circles doesn't end there though! Even though the cube factories closed down, the cubes didn't disappear - they just became a part of everyday life. You can find them in buildings, furniture and all sorts of other objects. The circle, on the other hand, is still around today but it's not as popular as it once was. Some people say that circles are too simple or outdated, but I think they're beautiful nonetheless.

i am very sussy

**Comment by nekoify** [comment]
no way

**Comment by Quantaviousdingleberry420** [comment]
add sex

**Reply by aeolus-1** [reply]
no

**Reply by nekoify** [reply]
yes

**Reply by Quantaviousdingleberry420** [reply]
> 

gay

**Comment by serifonium** [comment]
cube lore

**Comment by Quantaviousdingleberry420** [comment]
holy shit cube lore pog
